test/test_distributions.py

```diff
@@ -285,6 +285,8 @@ def get_sp_dist(jax_dist):
     T(_ImproperWrapper, constraints.positive, (), (3,)),
     T(dist.InverseGamma, np.array([1.7]), np.array([[2.0], [3.0]])),
     T(dist.InverseGamma, np.array([0.5, 1.3]), np.array([[1.0], [3.0]])),
+    T(dist.Kumaraswamy, 10.0, np.array([2.0, 3.0])),
+    T(dist.Kumaraswamy, np.array([1.7]), np.array([[2.0], [3.0]])),
```

Comment:
Could you add an example with `0. < conc0 or conc1 < 1.`?

Reply:
I just added the test.